TrackletAlgorithm/MatchProcessor.h

```diff
@@ -1651,19 +1653,14 @@ void MatchProcessor(BXType bx,
     validin = validmem;
     read_inmem<TrackletProjection<PROJTYPE>, TrackletProjectionMemory<PROJTYPE>, nMEM>(projdata, bx, read_address, imem, ipage, projin);
 
-    if (!projBuffNearFull){
-      
-      // read inputs
-      //validin = read_input_mems<TrackletProjection<PROJTYPE>, TrackletProjectionMemory<PROJTYPE>, nMEM, kNBits_MemAddr+1>
-      //(bx, mem_hasdata, numbersin, mem_read_addr, iMem, iPage, projin, projdata);
-
-      read_address =  mem_read_addr;
-      validmem = read_input_mems<TrackletProjection<PROJTYPE>, TrackletProjectionMemory<PROJTYPE>, nMEM, kNBits_MemAddr+1>
-	(bx, mem_hasdata, numbersin, mem_read_addr, iMem, iPage, imem, ipage);
+    // read inputs
+    //validin = read_input_mems<TrackletProjection<PROJTYPE>, TrackletProjectionMemory<PROJTYPE>, nMEM, kNBits_MemAddr+1>
+    //(bx, mem_hasdata, numbersin, mem_read_addr, iMem, iPage, projin, projdata);
```

Comment:
Remove this commented out code?

Reply:
Done, though I left the `// read inputs`, since that's still relevant.

Reply:
I think this looks fine. (Though some variable names could probably be improved - but this is my doing earlier...)